src/app/progress/powercaps.scss

```diff
@@ -0,0 +1,29 @@
+@use '../variables.scss' as *;
```

Comment:
We prefer using CSS modules for all new code - you can see examples elsewhere in the codebase.